apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/MapExpectationSamples.kt

```diff
@@ -0,0 +1,246 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class MapExpectationSamples {
+
+    @Test
+    fun toContainBuilder() {
```

Comment:
You forgot to add the `@sample` tag to `toContain` in mapExpectations.kt

Reply:
@kacmacuna it's still not there, see first function in mapExpectations

Reply:
Don't you mean this?
![Screenshot_36](https://user-images.githubusercontent.com/39156859/120605111-bce49900-c45e-11eb-8abb-70b311118f83.png)

Reply:
Hm... must have been tricked by GitHub and probably looked at an old revision. All good.